test/parallel/test-abortcontroller.js

```diff
@@ -4,8 +4,10 @@
 const common = require('../common');
 
 const { ok, strictEqual } = require('assert');
+const { Event } = require('internal/event_target');
```

Comment:
This line is unnecessary, as `Event` is available as a global constructor.

```suggestion
```

It also causes the test to fail without the `‑‑expose‑internals` flag.

Reply:
Event is currently not exposed globally as far as I know. Good catch on the flag - I'll add that.

Reply:
(Event is not exposed _at all_ at the moment because as you can tell - there are still a bunch of rough edges we need to iron out - I intended to work on this a lot more a few months ago and am only getting to it now)

Reply:
`Event` is exposed since <https://github.com/nodejs/node/pull/35496>:
<https://github.com/nodejs/node/blob/9ff25b19ac6c4b239aeeca5d9534523ff91eddf7/lib/internal/bootstrap/node.js#L143-L148>

Reply:
/ping @benjamingr Good to make this change or not so much?

Reply:
I have no strong feelings regarding this - I didn't realize that we expose event globally and totally missed it.

Reply:
@ExE-Boss very cool I missed this!